facebook_connector.py

```diff
@@ -68,8 +70,15 @@ def get_cleaned_fully_merged_messages(strip_html_content=True,
     me = chats.user
     addresses = set()
     messages = []
+    # Suppressing warning on BS4 that shows up when trying to parse non HTML content
+    # Some messages might contain HTML tags others might not and we don't want to display
+    # these warnings to the user
     warnings.filterwarnings("ignore", category=UserWarning, module='bs4')
```

Comment:
This is how it looks for me when I comment this line out
<img width="808" alt="screen shot 2017-03-25 at 5 24 13 pm" src="https://cloud.githubusercontent.com/assets/7909844/24326217/05074c46-1180-11e7-97d2-16499e5e2d94.png">